server/routes/bookedtoday/arrivals/searchforexisting/noExistingRecordsFoundController.ts

```diff
@@ -17,4 +21,26 @@ export default class NoMatchingRecordsFoundController {
       })
     }
   }
+
+  public submit(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const { id } = req.params
+
+      const arrival = await this.expectedArrivalsService.getArrival(id)
+      const searchData = State.searchDetails.get(req)
+
+      const data = {
+        firstName: convertToTitleCase(searchData.firstName),
+        lastName: convertToTitleCase(searchData.lastName),
+        dateOfBirth: searchData.dateOfBirth,
+        sex: arrival.gender,
```

Comment:
may be best not to set this? rather than default to potentially somebody elses value?

Reply:
Ah ok, so just leave the sex as undefined?